thermometer.py

```diff
@@ -1,63 +1,72 @@
+#!/usr/bin/python3
+#-*- coding: utf-8 -*-
+
 import threading
 import time
 import sys
-from random import *
+import random
 
 def main():
-    inputCommand = ""
+    input_command = ""
```

Comment:
I'm just putting this comment in so you can see how they work, also because this line is now redundant. The variable isn't set anywhere.